src/tiledb/cloud/vector_search/file_ingestion.py

```diff
@@ -0,0 +1,326 @@
+from typing import Dict, List, Optional, Sequence
+
+from tiledb.cloud import dag
+from tiledb.cloud.utilities import as_batch
+
+
+def ingest_files_dag(
```

Comment:
Need parameters for passing in OpenAI API tokens and using OpenAI embeddings.

Reply:
We don't have an OpenAI embeddings implementation yet for the object api.

Reply:
Added support for OpenAI embeddings